### DISCUSSION TITLE
Get Started with T1000-E Tracker | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1727] [Q&A]
# Get Started with T1000-E Tracker | Seeed Studio Wiki

Get Started with SenseCAP Card Tracker T1000-E for Meshtastic

https://wiki.seeedstudio.com/sensecap_t1000_e/

**Comment by Coopersmith-24601** [comment]
There needs to be better instructions for updating/installing the BOOTLOADER.

I've tried the both methods on MacOS and I'm getting nowhere.  I don't know if I'm confused about where things are located in directories or whether or not it's the serial port or what, but it's just confusing to the average user who is not intimately familiar with how it's supposed to be accomplished.

At this point my T1000-e is stuck in a boot loop with Meshtastic firmware.  I've tried erasing it and reinstalling that but every time it's failing.  I'm convinced my BOOTLOADER has become corrupted in some way so I need to refresh that, but the directions are simply indecipherable to me, an average user.

**Reply by Coopersmith-24601** [reply]
Did some research and got my problem solved updating the BOOTLOADER with the help of folks over at the main Meshtastic Discord.  But man, it stumped people over there for quite awhile too.

Yeah, the instructions on the SEEED Studio web site need to include a few more steps.  At least for MacOS users.  The documentation they have assumes knowledge which new users simply will not have like how to link up the downloaded boot loader.zip with the pip3 pathing and they need to be more helpful in helping the user figure out exactly how to discover which serial port their device might be linked to.

**Reply by mdaugustine1** [reply]
The instructions are awful. I don't know where to type the commands in on Windows.

**Reply by e010584** [reply]
I sure can't figure it out in Windows.  Anybody care to document what was learned?

**Reply by Jessie219-web** [reply]
Hi there, thanks for your feedback, we will optimize this part in more detail later.

**Reply by kawfey** [reply]
> Did some research and got my problem solved updating the BOOTLOADER with the help of folks over at the main Meshtastic Discord. But man, it stumped people over there for quite awhile too.
> 
> Yeah, the instructions on the SEEED Studio web site need to include a few more steps. At least for MacOS users. The documentation they have assumes knowledge which new users simply will not have like how to link up the downloaded boot loader.zip with the pip3 pathing and they need to be more helpful in helping the user figure out exactly how to discover which serial port their device might be linked to.

care to share with the class?

**Comment by M00NPIG** [comment]
Would be nice if the light blinked red when it has a message to retrieve.

**Reply by Jessie219-web** [reply]
the color cannot be changed currently, but it may be considered in future plans

**Comment by greghall-au** [comment]
The Mediatek AG3335 GNSS uses both L1 & L5, so is very capable.
https://github.com/lyusupov/SoftRF/wiki/Card-Edition is using it with his FlightAware system.
-
How can we cause the T1000-E to emit NMEA sentences by Bluetooth, so an ordinary connected device can use the precision location information?
-
Android/Developer options/'Mock' locations seems to be the awkward choice.
Is there a better way that works on 'ordinary' ppls phones?

**Reply by Jessie219-web** [reply]
Hi there, an open-source LoRaWAN version of the T1000-E will be available soon, then you can use the SDK to develop based on that.
Suggestion: instead of sending NMEA directly, you could broadcast the lat/lon via BLE after scanning the satellites, similar to a location beacon.

**Comment by morosakubek2** [comment]
Sending an nvme or mock location would be a game changer with this device.

**Comment by berlincount** [comment]
Just got my device and started charging it, accidentally putting it into DFU mode - is there any way to get out of DFU mode again _without_ letting the battery drain or connecting a computer? 🤔 pushing the button for 5+s doesn't do anything I could tell.

**Reply by Jessie219-web** [reply]
reboot the device to get out of DFU mode: hold the button and connect the charging cable.

**Reply by greghall-au** [reply]
Hi,
One of my T1000-e devices has shut down, (probably from flat battery) no LED on holding button down while connecting, nor in the range of tests that SEEED had me video myself doing.
They agreed that it is faulty and should be returned and replaced, but then insist that I deal with the original seller.
I can't find orig order, so I'm stuck with it.
I'm tempted to cut it open and disconnect the battery.
Any advances (or better suggestions) on that?
Greg

**Comment by ea1icz** [comment]
The source for the Lorawan T1000 is published, but are there instructions on how to recompile the sources for the Meshtastic T1000E to build a new firmware?

**Reply by Jessie219-web** [reply]
about the Meshtastic firmware, you may refer to: https://github.com/meshtastic/firmware/tree/master

**Comment by Salamimonster** [comment]
I accidentally turned off bluetooth in the settings of one of my T1000-E devices.
How to turn it on again?

**Reply by 4a4b3837** [reply]
USB > https://client.meshtastic.org/  > serial connection

**Reply by Salamimonster** [reply]
It worked. Thank you.

**Comment by dselzle** [comment]
Is there any way to disable the startup melody?

**Reply by Zeke5579** [reply]
You can disable the buzzer use the Meshtastic app. However, when you receive a message, the buzzer will also not work.

**Reply by 4a4b3837** [reply]
The appsettings effect only the message notification. The startup melody bleeps any way.

**Comment by toddkrein** [comment]
Does triple-button press (while running 2.6 meshtastic) toggle GPS or send a ping? I need a way to send a position message from my T1000-E.

**Reply by 4a4b3837** [reply]
Newest Firmware:

3x clicks toggle GPS on/off
2x clicks Broadcast your Position in channel 0

**Comment by alexl83** [comment]
I'd like to use e-lux sensor with meshatstic's "detection sensor module" e.g. when light is detected an alert is triggered
Can you advise how to configure it?

The same I'd like to achieve with accelerometer; when movement is detected an alert is triggered

**Reply by 4a4b3837** [reply]
First of All you need to know that the Detektion Module is Hidden in FW >2.5.20.
So you have to go with a Dinosaur of firmware, because of the developers' opinion that it is user-friendly to decide what the user wants as a stranger.
Unfortunately, I can't help you with the lux meter. Detection sensor at touch of the button, I use every day. Of course with FW 2.5.20.

**Comment by greghall-au** [comment]
and the GNSS system is Mediatek AG3335, L2 & L5 !
Has anyone succeded in getting the T1000E to spit out NMEA via Bluetooth (or perhaps extract it from the Meshtastic app for use elsewhere), so we can use the extraodinary precision of L5?
https://www.airoha.com/products/p/VXKPfHI9iDCvsRWN

**Reply by nfs0619** [reply]
Currently, retrieving GNSS data via Bluetooth in the Meshtastic app is not yet supported.
If you’d like to access the T1000-E’s GNSS data, you can do so through the serial interface. Please see the configuration details here:
https://meshtastic.org/docs/configuration/module/serial/#serial-module-config-values

**Comment by marcusobrien** [comment]
Where are the instructions for setting up a channel for multiple Meshtastic devices toall communicate with each other like a group chat ? I'd like the instructions for FW version 2.6.11 for the Sensecap T1000E.

Also some feedback - some of these instructions go out of date when new firmware is released, or a new Mobile Application is released. All instructions should be updated when new releases are made, if necessary.

**Reply by nfs0619** [reply]
Hello, thank you for your feedback! For setting up a channel, you can refer to the instructions in this document:
https://meshtastic.org/docs/software/android/usage/#setup-a-channel

If you encounter any other issues during use, please feel free to reach out to our technical support team at **[sensecap@seeed.cc](mailto:sensecap@seeed.cc)** — they can provide you with faster and more detailed guidance.

**Comment by vatrubin** [comment]
I got one T1000-E and it works fine but battery indicator always on 0%.
Sometimes after restart voltage above 0V. 0.77V, 0.01V but most of time it is 0V. 
Is it hardware issue?
Can i fix it with ADC multiplier override?
Use last firmware beta: 2.7.15.567

**Reply by nfs0619** [reply]
Apart from the battery level display, please let us know whether there are any other abnormal behaviors and whether the device can power on and work normally. If there are no other functional issues, we suspect the battery may need calibration. Please fully discharge the device and then charge it continuously for more than 8 hours, repeat this process twice, and observe whether the battery level display returns to normal. Please also make sure you are not using a fast-charging power adapter. If there are any other abnormal functions, please contact us via email at **[techsupport@seeed.io](mailto:techsupport@seeed.io)** as soon as possible.
